spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1489,10 +1489,31 @@ def name_id_version(format_urn)
             expect(response.status).to eq(200)
             expect(@analytics).to have_logged_event('SAML Auth', hash_including(analytics_hash))
           end
+
+          context 'Certificate does not match because of order bug' do
+            let(:encryption_cert_matches_matching_cert) { false }
```

Comment:
Minor suggestion - `encryption_cert_matches_matching_cert` seems awkward to me. May I suggest `encryption_cert_matches_provided_cert` or similar?

Reply:
hm that makes sense, it's a bit awkward, but it's referencing the key in the analytics, and that is referencing specific methods. (and i think it more precise than "provided" cert. it's testing if the cert that is being returned as "matching" is the same cert as is being used in the encryption).

Reply:
Ah, that makes sense!

Reply:
exactly!